pkg/runtime/service_controller.go

```diff
@@ -34,18 +34,37 @@ type ServiceController struct {
 	// rmFactories is a map of resource manager factories, keyed by the
 	// GroupKind of the resource managed by the resource manager produced by
 	// that factory
-	rmFactories map[string]types.AWSResourceManagerFactory
+	rmFactories map[string]acktypes.AWSResourceManagerFactory
 	// reconcilers is a map containing AWSResourceReconciler objects that are
 	// bound to the `controller-runtime.Manager` in `BindControllerManager`
-	reconcilers []types.AWSResourceReconciler
+	reconcilers []acktypes.AWSResourceReconciler
 	// log refers to the logr.Logger object handling logging for the service
 	// controller
 	log logr.Logger
 }
 
+type ServiceControllerCloudCfg interface {
+	// AccountID provides AccountID for service controller to create the service object
+	AccountID() string
+	// Region for the service controller to create the service object
+	Region() string
+}
+
+type defaultControllerCfg struct {
+	cfg Config
+}
+
+func (c defaultControllerCfg) AccountID() string {
+	return c.cfg.AccountID
+}
+
+func (c defaultControllerCfg) Region() string {
+	return c.cfg.Region
+}
```

Comment:
I don't believe you need the above `interface`. Just use the concrete `pkg/config.Config` struct.